addons/web/static/src/scss/navbar.scss

```diff
@@ -1,4 +1,11 @@
-
+body.o_is_superuser .o_menu_systray {
+    background: repeating-linear-gradient(
```

Comment:
Lol, I actually like your art :-)
![screenshot from 2018-09-27 12-46-15](https://user-images.githubusercontent.com/2039870/46142081-10d26500-c256-11e8-94cc-4d71ec870b2f.png)

It's not meant to be for everyday use, and I find it quite effective.
@stefanorigano what do you think?